website/layouts/docs.jsx

```diff
@@ -1,32 +1,35 @@
+import Head from 'next/head'
+import Link from 'next/link'
 import DocsPage from '@hashicorp/react-docs-page'
+import { createMdxProvider } from '@hashicorp/nextjs-scripts/lib/providers/docs'
 import order from '../data/docs-navigation.js'
 import { frontMatter } from '../pages/docs/**/*.mdx'
-import { MDXProvider } from '@mdx-js/react'
 import Placement from '../components/placement-table'
-import Head from 'next/head'
-import Link from 'next/link'
 
-const DEFAULT_COMPONENTS = { Placement }
+const MDXProvider = createMdxProvider({
```

Comment:
@notnoop  This new wrapper includes the `<EnterpriseAlert />` and `<Tabs />` components by default so you can use them within the `/docs` pages now without explicitly importing them.